lib/view_component/base.rb

```diff
@@ -95,7 +95,12 @@ def render_in(view_context, &block)
       before_render
 
       if render?
-        render_template_for(@__vc_variant).to_s + _output_postamble
+        custom_variant = [@__vc_variant]
+        if defined?(I18n.locale) && I18n.locale != :en
```

Comment:
I'm not sure about the `!= :en` part. What is the goal? What about using `I18n.default_locale` instead?

Reply:
I'm still thinking about this.

The problem is that files without localized filename (ie "mycomponent.html.erb") generates a `call` function. If I consider `I18n.default_locale` or `I18n.locale` it will create a function `call_{locale}` that will break other things...  :thinking: